src/styles/app.scss

```diff
@@ -120,6 +120,10 @@ code {
   overflow: auto;
 }
 
+ul a code {
```

Comment:
this seems like quite a specific selector 😅 

while this would probably fix the issue for the specific blog post you took a screenshot of, I assume it won't work for `ol` lists, for `code` blocks that are not links, and for `code` blocks that are not in lists.

do you think we can find a solution that would work for all of these cases? :)

Reply:
Fair point, I agree with you. That was a quick fix. I'll make a note and tackle this properly when I have more time. Meanwhile, I'll close the PR.